scripts/write_deployinst_input.py

```diff
@@ -201,43 +200,86 @@ def reactor_render(list, template, mox_template, output_file):
         name of output file
+    is_cyborg: bool
+        if True, uses Cyborg templates
 
     Returns
     -------
     The reactor section of cyclus input file
 
     """
-
+    if is_cyborg:
+        pwr_template = read_template('../templates/pwr_template_cyborg.xml.in')
+        mox_reactor_template = read_template(
+            '../templates/mox_template_cyborg.xml.in')
+        candu_template = read_template(
+            '../templates/candu_template_cyborg.xml.in')
+    else:
+        pwr_template = read_template('../templates/pwr_template.xml.in')
+        mox_reactor_template = read_template(
+            '../templates/mox_template.xml.in')
+        candu_template = read_template('../templates/candu_template.xml.in')
     for data in list:
         # BWRs have different fuel assembly size, assembly per core and batch
-        name = data['reactor_name'].decode('utf-8').title()
+        name = data['reactor_name'].decode('utf-8')
         start = name.find('(')
         end = name.find(')')
         if start != -1 and end != -1:
             name = name[:start]
+        # if BWR,
         if data['type'].decode('utf-8') == 'BWR':
-            reactor_body = template.render(
+            reactor_body = pwr_template.render(
                 country=data['country'].decode('utf-8'),
+                type=data['type'].decode('utf-8'),
                 reactor_name=name,
                 assem_size=180,
                 n_assem_core=int(round(data['capacity']/1000 * 764)),
                 n_assem_batch=int(round(data['capacity']/3000 * 764)),
                 capacity=data['capacity'])
+        # if PHWR or CANDU,
+        elif data['type'].decode('utf-8') == 'PHWR' or 'CANDU' in data['type'].decode('utf-8'):
+            reactor_body = candu_template.render(
+                country=data['country'].decode('utf-8'),
+                type=data['type'].decode('utf-8'),
+                reactor_name=name,
+                assem_size=int(80000/473),
+                n_assem_core=int(round(data['capacity']/500 * 473)),
+                n_assem_batch=60,
+                capacity=data['capacity'])
         # if French PWR, use mox template for mox reactor
         elif data['type'].decode('utf-8') == 'PWR' and data['country'].decode('utf-8') == 'France':
-            reactor_body = mox_template.render(country=data['country'].decode('utf-8'),
+            reactor_body = mox_reactor_template.render(country=data['country'].decode('utf-8'),
+                                                       reactor_name=name,
+                                                       type=data['type'].decode(
+                                                           'utf-8'),
+                                                       assem_size=446,
+                                                       n_assem_core=int(
+                round(data['capacity']/1000 * 193)),
+                n_assem_batch=int(
+                round(data['capacity']/3000 * 193)),
```

Comment:
What is this calculation about ? Also, this will be completely wrong in python 2.